File: Domain/Lecturer.cs
namespace Domain;

public class Lecturer : AppUser
{
    public DepartmentSubject DepartmentSubject { get; set; }
    public Faculty Faculty { get; set; }
    public ICollection<Student> Students { get; set; }
    public int InstructorStatus { get; set; }
    public int MaxStudentsNumber { get; set; }
    public string Education { get; set; }
}

File: client-app/src/app/models/lecturer.ts
export interface Lecturer {
    id: string;
    displayName: string;
    email: string;
    birthday?: Date;
    bio: string;
    sex?: boolean;
    instructorStatus: number;
    lecturerStatus: string;
    phoneNumber: string;
    studentCount: number;
    maxStudentsNumber: number;
    education: string;
    departmentSubjects: string;
    faculty: string;
    createdDate: Date;
}